website/components/layout.js

```diff
@@ -8,14 +9,72 @@ import Navbar from "react-bootstrap/Navbar";
 import Nav from "react-bootstrap/Nav";
 import NavDropdown from "react-bootstrap/NavDropdown";
 
+import Collapse from "react-bootstrap/Collapse";
 import Form from "react-bootstrap/Form";
 import FormControl from "react-bootstrap/FormControl";
 import Button from "react-bootstrap/Button";
 
-export default function Layout({ showTOC = "True", children }) {
+function makeChapter(chapter) {
+  return (
+    <NavDropdown.Item key={chapter.file}>
+      <Link href={`/chapters/${chapter.file}`}>
+        <span>
+          {chapter.chapterNumber}. {chapter.title}
+        </span>
+      </Link>
+    </NavDropdown.Item>
+  );
+}
+
+function makeCaseStudy(caseStudy) {
+  const [open, setOpen] = useState(false);
+
+  function onClick(evt) {
+    setOpen(!open);
+    evt.preventDefault();
+    evt.stopPropagation();
+  }
+
+  return (
+    <div key={caseStudy.caseStudy}>
+      <NavDropdown.Item>
+        <button
+          onClick={onClick}
+          aria-controls={`case-study-${caseStudy.caseStudy}`}
+          aria-expanded={open}
+          className={styles.unbutton}
+        >
+          {" "}
+          🌟 {caseStudy.title}
```

Comment:
The color of the star kinda clashes with the background imo, I would do no bullet point and make the text not italic, unless the star has signficance.